streaming-binlog/pom.xml

```diff
@@ -41,6 +41,8 @@
     <version.kafka>2.4.0</version.kafka>
     <version.plugin.docker>0.31.0</version.plugin.docker>
     <version.plugin.shade>3.2.1</version.plugin.shade>
+    <maven-jar-plugin.version>3.0.2</maven-jar-plugin.version>
+    <maven-shade-plugin.version>3.1.0</maven-shade-plugin.version>
```

Comment:
We should probably move these two plugin config to `pluginManagement` of the root `pom.xml` but this can be done in future too, if you prefer.